src/main/java/org/mockito/internal/runners/RunnerFactory.java

```diff
@@ -16,9 +21,34 @@
  */
 public class RunnerFactory {
 
+    /**
+     * Creates silent runner implementation
+     */
     public RunnerImpl create(Class<?> klass) throws InvocationTargetException {
+        return create(klass, new Supplier<MockitoTestListener>() {
+            public MockitoTestListener get() {
+                return new NoOpTestListener();
+            }
+        });
+    }
+
+    /**
+     * Creates strict runner implementation
+     */
+    public RunnerImpl createStrict(Class<?> klass) throws InvocationTargetException {
+        return create(klass, new Supplier<MockitoTestListener>() {
+            public MockitoTestListener get() {
+                return new MismatchReportingTestListener(new ConsoleMockitoLogger());
+            }
+        });
+    }
+
+    /**
+     * Creates runner implementation with provided listener supplier
+     */
+    public RunnerImpl create(Class<?> klass, Supplier<MockitoTestListener> listenerSupplier) throws InvocationTargetException {
         try {
-            return new RunnerProvider().newInstance("org.mockito.internal.runners.SilentJUnitRunner", klass);
+            return new RunnerProvider().newInstance("org.mockito.internal.runners.SilentJUnitRunner", klass, listenerSupplier);
```

Comment:
Yes I supposed this was that, but since the class is in the same package it feels not needed. And may complicate a tad more renaming.